contrib/development/update-gold

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+cur_dir=${PWD}
+if [[ ! -x ${cur_dir}/unittests ]]; then
+    printf -- "ERROR: ${cur_dir} is not where a 'unittests' executable lives.\n" >&2
+    exit 1
+fi
+
+export _GF="/tmp/gold_files.$$"
+
+trap "rm -f ${_GF}" INT EXIT
+
+function _do_update {
+    > ${_GF}
+    ls -1 gold/test-${1}.txt gold/${1}.txt gold/*/test-${1}.txt gold/*/${1}.txt tests/*/test-${1}.txt tests/*/${1}.txt > ${_GF} 2> /dev/null
```

Comment:
Without being completely certain exactly how and where your ${1} pattern might appear in nested directory filenames... would this be more efficient as a `find gold -name \*${1}.txt` ???

Reply:
It might be.  I originally had this working for one kind of unit test structure, but then we kept making additional changes to and I just added the other names and locations.